README.md

```diff
@@ -68,10 +68,6 @@ services:
 
 See the [docker-compose.yml](docker-compose.yml) file for details.
 
-If you have modified the source code, then you may want to build the docker image locally first:
-
-    docker build -t commonworkflowlanguage/cwlviewer .
-
```

Comment:
Why was this removed? It worked for me..

Reply:
> Thank you @obasekiosa for this PR.
> 
> Why was an empty `docker-compose.override.yml` added?

The empty file was a mistake.



> Why was this removed? It worked for me.

It seems to be doing the exact same thing as the override file and for some reason, it didn't work for me.
Changes I made did not show up until I used the override file.
Also, the instruction seems to be a duplicate of the one above it. What to do after a change has been made to the source code.